examples/echoxp.py

```diff
@@ -3,34 +3,27 @@
 from mininet.log import setLogLevel, info
 from mininet.cli import CLI
 from mininet.link import Link
-from mininet.net_xia import Mininet_xia
-from mininet.node_xia import Node_xia
+from net import Mininet
+from node import XIAHost
 
 def topology():
-	"a V-topology network for net-echo experiment"
+	"A topology for net-echo experiment"
 
-	net = Mininet_xia(controller=None)
+	net = Mininet(controller=None)
 
 	info("*** creating nodes\n")
-
-
+	xia1 = net.addHost('xia1', cls=XIAHost , hid='foo')
+	xia2 = net.addHost('xia2', cls=XIAHost , hid='bar')
```

Comment:
This way of passing in XIDs is not very flexible. What if we wanted to assign multiple HIDs to the host?

Reply:
It's actually quite flexible. You may provide a string of multiple private key filenames(comma separated) eg hid='pk1,pk2,pk3' and the host will have HIDs for the 3 corresponding filenames